src/components/Header/Tools.tsx

```diff
@@ -241,7 +247,12 @@ const Tools = () => {
     },
     {
       title: betaSwitcherTitle,
-      onClick: () => (window.location.href = switchRelease(isBeta(), location.pathname, previewEnabled)),
+      onClick: () => {
+        if (!LOCAL_PREVIEW) {
```

Comment:
@Hyperkid123 this `onClick` is identical to the `onChange` below. Can we de-dupe them in a helper function?